tests/DocumentationExamplesTest.php

```diff
@@ -1713,7 +1711,7 @@ public function testVersionedApiMigration(): void
         try {
             $count = $db->sales->count();
         } catch (\MongoDB\Driver\Exception\CommandException $e) {
-            echo json_encode($e->getResultDocument());
+            echo json_encode($e->getResultDocument(), JSON_THROW_ON_ERROR);
```

Comment:
Since this appears within a documentation example, I think we should use a leading backslash.

```suggestion
            echo json_encode($e->getResultDocument(), \JSON_THROW_ON_ERROR);
```

Reply:
For the purpose of documentation, we should not add this option : it complicates the example without adding anything useful. The JSON returned by mongo is always valid. 
The rector rule can be disabled for this file.

Reply:
I agree. I excluded the file from that rector.

Reply:
Ah, I didn't realize rector was adding this. I agree it's not needed for example code.